library/src/main/res/values/strings.xml

```diff
@@ -37,16 +37,16 @@
     <string name="chucker_tab_network">Network</string>
     <string name="chucker_tab_errors">Errors</string>
     <string name="chucker_network_tutorial">This tab displays network requests recorded with the ChuckerInterceptor. Add the interceptor to your OkHttp client to reap the benefits.</string>
-    <string name="chucker_errors_tutorial">This tab displays error recorded with the ChuckerCollector.onError method. Call this method when an error is thrown to check your stacktrace live.</string>
-    <string name="chucker_networks_notification_category">Chucker network requests</string>
-    <string name="chucker_errors_notification_category">Chucker throwables</string>
-    <string name="chucker_share_error_title">Share error details</string>
+    <string name="chucker_throwable_tutorial">This tab displays throwables recorded with the ChuckerCollector.onError method. Call this method when an error is thrown to check your stacktrace live.</string>
```

Comment:
nit: `Throwables` (uppercase) as it's a reference to subclasses of that class.

Reply:
But I thought that it is a self-made word, since we don't have a word for multiple `Throwable` items.

Reply:
yeah makes sense :)